main.cpp

```diff
@@ -1,11 +1,22 @@
-#include "mainwindow.h"
+#include "MainWindow/MainWindow.h"
 #include <QApplication>
 
+#include "GlobalNames.h"
+
 int main(int argc, char *argv[])
 {
-    QApplication a(argc, argv);
-    MainWindow w;
-    w.show();
+    //Q_INIT_RESOURCE(spacePaint);
+
+    QApplication Application(argc, argv);
+
+    QCoreApplication::setApplicationName(GLOBAL_STRINGS::PROGRAM_INFO::PROGRAM_NAME);
+    QCoreApplication::setOrganizationDomain(GLOBAL_STRINGS::PROGRAM_INFO::ORGANIZATION_NAME);
+    QCoreApplication::setApplicationName(GLOBAL_STRINGS::PROGRAM_INFO::PROGRAM_VERSION);
+
```

Comment:
I try divide and group code invoke in logical way. In my opinion, if there are not any of blank lines, then it will be hard to read.
If you'r have concrete in mind, please write it.